### ISSUE TITLE
What happened?

### ISSUE THREAD

**Issue by IceRaider** [CraftBB/CraftBB#1]
What happened to this project? is it still in development, can I download it?

**Comment by CraftBB** [comment]
Yes, it's still in development, I just haven't had time to work on it
recenlty and I'm going to be coding it from ground up this time, instead of
using a framework and composer
